utils/wasm-gen/src/config.rs

```diff
@@ -217,3 +218,69 @@ impl ConfigsBundle for StandardGearWasmConfigsBundle {
         (gear_wasm_generator_config, selectable_params)
     }
 }
+
+pub struct RandomizedGearWasmConfigBundle {
```

Comment:
after defining `new_artbitrary` for `Randomized` (maybe even implementing `Arbitrary`, not sure) config, please test it the same way as `Standard` config was tested for generating wasms that are always valid for `gear_core::Code` instantiation by `Code::try_new` (see tests)